content/site.json

```diff
@@ -18,7 +18,8 @@
     ],
     "warning": {
       "main": "We are in beta. Please familiarize yourself with [the market](https://oceanprotocol.com/technology/marketplaces), [the risks](https://blog.oceanprotocol.com/on-staking-on-data-in-ocean-market-3d8e09eb0a13), and the [Terms of Use](/terms).",
-      "polygonPublish": "Only republish data sets with a pool from ETH Mainnet into Polygon/Matic if the liquidity is **less than or equal to 1000 OCEAN in the original pool**. Doing otherwise will lead to [purgatory](https://github.com/oceanprotocol/list-purgatory) for the data set in Polygon/Matic."
+      "polygonPublish": "Only republish data sets with a pool from ETH Mainnet into Polygon/Matic if the liquidity is **less than or equal to 1000 OCEAN in the original pool**. Doing otherwise will lead to [purgatory](https://github.com/oceanprotocol/list-purgatory) for the data set in Polygon/Matic.",
+      "graphNotSynched": "Graph out of sync, possible data lack."
```

Comment:
right, good point getting copy in here. Quickly hacked it into `App.tsx` but feel free to find some way to inject the dynamic values into the text somehow

Reply:
so either we figure out how to get the copy currently in `App.tsx` into here, or we remove this copy line

Reply:
ok, I'll look into it

Reply:
Couldn't find a way for that, so we can remove the line and keep it as it is for now